packages/yourloops/components/patient-list/utils/sort-comparators.util.ts

```diff
@@ -98,3 +101,17 @@ export const sortByMonitoringAlertsCount: PatientSortComparator = (patient1: Pat
   }
   return 0
 }
+
+export const sortByLastDataUpdate: SortComparator = (date1AsString: string, date2AsString: string): number => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏We could mutualize the date management with the `sortByDate` comparator already in this file. It would require to modify the method `getLastUploadDate()` to return only the raw date, and put the formatting part in another method to call in the `valueFormatter` of the column. I think it's doable, but not trivial so I let you be the judge on what to do 😄

Reply:
I don't intend to do it. :wink: